src/airport.js

```diff
@@ -1 +1,79 @@
+class Airport {
+    #capacity
+    #planesList;
+    #weather;
 
+    constructor(capacity = 5) {
```

Comment:
The code follows clear, well-organised and concise structure.
Attempted to minimise the use of if statements